src/lab4.py

```diff
@@ -0,0 +1,68 @@
+class Farm:
+
+    pub_number = 100
+    pub_string = 'Info'
+
+    def __init__(self, location, animals_qty, vent_power, milk):
```

Comment:
var 'vent_power' isn't used through the whole code

Reply:
this var is used in output, I don't understand what do I have to do